client/components/quizList/QuizListCard.jsx

```diff
@@ -49,11 +51,15 @@ const  QuizListCard = ({ quiz, loggedInUser }) => {
           </Typography>
         </CardContent>
         <CardContent>
-        <ChallengeFriend
-          loggedInUser={loggedInUser}
-          friends={friends}
-          link={`localhost:3000/quiz/${quiz.id}`}
-        />
+        {loggedInUser.username ?
+          <ChallengeFriend
+            loggedInUser={loggedInUser}
+            friends={friends}
+            link={`localhost:3000/quiz/${quiz.id}`}
```

Comment:
This is just a placeholder correct? It'll eventually be the deployed site's link?

Reply:
Mike confimed